bigquery/google/cloud/bigquery/client.py

```diff
@@ -1229,7 +1241,11 @@ def list_rows(self, table, selected_fields=None, max_results=None,
 
         :type page_token: str
         :param page_token: (Optional) Token representing a cursor into the
-                           table's rows.
+                           table's rows.  If not passed, the API will return
+                           the first page of the rows. The token marks the
+                           beginning of the iterator to be returned and the
+                           value of the page_token can be accessed at
+                           next_page_token of the HTTPiterator.
```

Comment:
`list_rows` uses a more specialized class than `HTTPIterator`. See the return type: 

```
:class:`~google.cloud.bigquery.table.RowIterator`
```